glib-macros/src/enum_derive.rs

```diff
@@ -3,24 +3,31 @@
 use heck::{ToKebabCase, ToUpperCamelCase};
 use proc_macro2::TokenStream;
 use proc_macro_error::abort_call_site;
-use quote::{quote, quote_spanned};
+use quote::{quote, quote_spanned, ToTokens};
 use syn::{punctuated::Punctuated, spanned::Spanned, token::Comma, Data, Ident, Variant};
 
 use crate::utils::{crate_ident_new, gen_enum_from_glib, parse_nested_meta_items, NestedMetaItem};
 
-// Generate glib::gobject_ffi::GEnumValue structs mapping the enum such as:
+// If `wrap` is `false`, generates glib::gobject_ffi::GEnumValue structs mapping the enum such as:
```

Comment:
For simplification, can't we generate the wrapped one always if that's needed somewhere? The non-wrapped one can be directly pulled out of it again if needed.